src/components/Book.js

```diff
@@ -1,16 +1,46 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { CircularProgressbar } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
 
 const Book = ({ book, handleRemoveBook }) => {
   const handleBookRemove = () => {
     handleRemoveBook(book);
   };
+  const percentage = 66;
   return (
-    <tr key={book.id} id={book.id}>
-      <td>{book.title}</td>
-      <td>{book.category}</td>
-      <td><button type="button" onClick={handleBookRemove} className="btn btn-danger">Delete</button></td>
-    </tr>
+    <div className="card">
+      <div className="row card-inside">
+        <div className="col-12 col-sm-3 col-md-4">
+          <p className="gender-book">{book.category}</p>
+          <p className="title-book">{book.title}</p>
+          <p className="author-book">JRR Tolkien</p>
+          <div className="row">
+            <div className="col-12 col-lg-5">
+              <a className="book-option" href="/">Comments</a>
+            </div>
+            <div className="col-12 col-lg-4">
+              <a className="book-option" href="/">Remove</a>
+            </div>
+            <div className="col-12 col-lg-3">
+              <a className="book-option" href="/">Edit</a>
+            </div>
+          </div>
+        </div>
+        <div className="col-6 col-sm-3 col-md-2 mt-4">
+          <CircularProgressbar value={percentage} />
+        </div>
+        <div className="col-12 col-sm-3 col-md-2">
```

Comment:
### Great job implementing the progress bar, let us do better by:

- [ ] Adjusting its size to match the design
- [ ] The percentage value can be a random number between 0 - 100 other than having a static value for all books.
- [ ] Ensure the gradient color matches the design

Reply:
- Regarding a random number for percentage, I invite you to read the more carefully the [documentation](https://microverse.pathwright.com/library/react-redux-v11/184438/path/step/106376032/). To create random numbers is just implement the functionality in the percentage number, I just put only one because the required on this milestone is just a placeholder.
![image](https://user-images.githubusercontent.com/20563252/125807343-efc673f6-fbd6-40bc-ae00-da7d76299b61.png)

Reply:
I think @vichuge is right here, Elbie. I would give this a pass as the requirement is not mandatory. So we will safely ignore this requirement.

Reply:
The point that I checked is what I change on my project.